game_runner.rb

```diff
@@ -0,0 +1,21 @@
+class GameRunner
+  def grid_template
```

Comment:
This shouldn't be here now! This is code you don't need because you are implementing anything yet.